### PR TITLE
ENH: update start trajectory processing

### PR DESCRIPTION
**Pull request by lcdesilva**
Modified "start trajectory" to follow the same style as in #5

### CODE REVIEW

File: startup/95-plans.py [hunk 1]
```diff
@@ -120,29 +120,11 @@ def inner():
         yield from bps.abs_set(mono1.start_trajectory, '1', wait=True)
 
         # this should be replaced by a status object
+        # Updated below method. Should we remove the comment above?
```
Comment (mrakitin) [inline startup/95-plans.py]: Yes, please remove the comment above and this comment ;)

File: startup/95-plans.py [hunk 2]
```diff
@@ -120,29 +120,11 @@ def inner():
...
-                    break
-                is_running = ret['mono1_trajectory_running']['value']
-
-                if is_running:
-                    yield from bps.sleep(.05)
-                else:
-                    break
 
+            yield from bps.mv(mono1, 'start')
+            yield from bps.sleep(delay)
```
Comment (mrakitin) [inline startup/95-plans.py]: I think we don't have the `delay` kwarg in the function. Please remove the line above.

File: startup/95-plans.py (hunk 1 above)
Comment (lcdesilva) [reply startup/95-plans.py]: Ok. I can do that in the new branch that I created. And then do another PR. Then what should I do about this PR? What does the "Resolve conversation" button do?

File: startup/95-plans.py (hunk 2 above)
Comment (lcdesilva) [reply startup/95-plans.py]: From both "prepare" and here in "start". I believe "prepare" also have the same line

Comment (mrakitin) [reply startup/95-plans.py]: 'prepare' function has it, but not this 'start' one. Olease check the definition of the function and the signature.

File: startup/95-plans.py (hunk 1 above)
Comment (mrakitin) [reply startup/95-plans.py]: No, just make changes to this exact branch and push them to the PR - the changes will be reflected immediately. No need for a new PR. I guess, 'resolve conversation' means we all are satisfied with the changes, but may be wrong, I never used it.

File: startup/95-plans.py (hunk 2 above)
Comment (lcdesilva) [reply startup/95-plans.py]: Got it.

File: startup/95-plans.py (hunk 1 above)
Comment (lcdesilva) [reply startup/95-plans.py]: Just did the "git push" after making the changes

Comment (lcdesilva) [reply startup/95-plans.py]: I did not commit. Do I have to? Going to check git for updates for PR 5.

Comment (lcdesilva) [reply startup/95-plans.py]: I mean PR #6

Comment (mrakitin) [reply startup/95-plans.py]: Push will do nothing before you commit. Sure, please commit first.

Comment (lcdesilva) [reply startup/95-plans.py]: Done.

Comment (mrakitin) [reply startup/95-plans.py]: OK, this looks good now.

File: startup/20-motors.py [hunk 3]
```diff
@@ -101,6 +102,24 @@ def callback(value, old_value, **kwargs):
...
+            def callback(value, old_value, **kwargs):
+                if int(round(old_value)) == 1 and int(round(value)) == 0:
+                    if self._starting or self._starting is None:
+                        self._starting = False
+                        return True
+                    else:
+                        self._starting = True
+                return False
+
+            #Need to double check the logic
```
[the remark this answers is not in the document]
Comment (lcdesilva) [reply startup/20-motors.py]: Sure. I guess will then automatically update PR #6 and needs approval again?

[the remark this answers is not in the document]
Comment (mrakitin) [reply startup/20-motors.py]: No re-approval is formally needed. Thanks for the update. Merging.

**Comment by lcdesilva** [comment]
I will go ahead with the merge request.

Ahem.... Maksim. You beat me to it :-)

**Comment by lcdesilva** [comment]
I will pull it from WS1 around 11.30 and do testing. If it breaks, I will do "git log" and check out the old version. I'd appreciate a bit of instruction here.

**Comment by mrakitin** [comment]
@lcdesilva, it's a good practice to not merge your own PRs unless you work on your personal project without collaborators. Better to have someone doing it.

**Comment by mrakitin** [comment]
Instructions: `git log` -> pick the commit hash you want to revert to -> `git checkout <hash>` -> create a new branch locally just in case (`git checkout -b old_master` or other better name) .

**Pull request merged**